노현호/스택_큐/같은 숫자는 싫어.js

```diff
@@ -0,0 +1,17 @@
+function solution(arr)
+{
+    let a = 0
+    let answer = []
+    for(let i = 0 ; i <arr.length; i++){
+        if(i+1 <= arr.length){
+            if(arr[i] === arr[i+1]){
+                continue
+            } 
+            else{
+                answer.push(arr[i])
+            }
```

Comment:
아래처럼 코드를 단축할 수 있을 것 같아요!
```suggestion
            if(arr[i] !== arr[i+1]){
                answer.push(arr[i])
            } 
```